assets/sass/components/global/_googlesitekit-side-sheet.scss

```diff
@@ -0,0 +1,59 @@
+/**
+ * Side Sheet styles.
+ *
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-plugin {
+
+	.googlesitekit-side-sheet {
+		background-color: $c-surfaces-surface;
+		bottom: 0;
+		overflow: hidden;
+		position: fixed;
+		right: 0;
+		top: 0;
+		transform: translateX(100%);
+		transition: transform 1s;
+		width: 100%;
+		z-index: $z-index-sk-side-sheet;
+
+		@media (min-width: $bp-tablet) {
+			width: 384px;
+		}
+
+		body.admin-bar & {
+			top: 46px;
+
+			@media (min-width: $bp-wpAdminBarTablet) {
+				top: 32px;
+			}
+		}
+	}
+
+	.googlesitekit-side-sheet--open {
+		transform: translateX(0);
+	}
+
+	.googlesitekit-side-sheet-overlay {
+		background-color: rgba(0, 0, 0, 0.6);
```

Comment:
Do we not have a variable for this?

Reply:
Doesn't look like we have a variable for this specific one, but I've created one now. Let me know if that looks good, thanks!